include/arch/posix/linker.ld

```diff
@@ -25,19 +25,28 @@ SECTIONS
 /* Located in generated directory. This file is populated by the
  * zephyr_linker_sources() Cmake function.
  */
-#include <snippets-rom-start.ld>
+SECTION_PROLOGUE(rom_start,,)
+{
+	#include <snippets-rom-start.ld>
+} GROUP_LINK_IN(ROMABLE_REGION)
 
 #include <linker/common-rom.ld>
 
 /* Located in generated directory. This file is populated by the
  * zephyr_linker_sources() Cmake function.
  */
-#include <snippets-rodata.ld>
+SECTION_PROLOGUE(rodata,,)
```

Comment:
There's `_RODATA_SECTION_NAME` which can be used here.

Also `_DATA_SECTION_NAME` and `_NOINIT_SECTION_NAME`

Reply:
Fixed, now using `_RODATA_SECTION_NAME`, `_DATA_SECTION_NAME`, and `_NOINIT_SECTION_NAME`